parastell/invessel_build.py

```diff
@@ -346,6 +346,33 @@ def export_cad_to_dagmc(self, dagmc_filename="dagmc", export_dir=""):
 
         model.export_dagmc_h5m_file(filename=str(export_path))
 
+    def export_invessel_component_mesh(
```

Comment:
Since this is part of the `InVesselBuild` class, I don't think it's necessary to specify in the name that this method is for in-vessel components. We may want to shorten the name to something like `export_mesh` for brevity and also to match the method name we use for the magnets.

Reply:
Yep I agree 'invessel' here is redundant. Yet, I think I would keep 'components' in the name of the method, since it meshes, and exports, specific components and not the whole radial build. Something like `export_component_mesh` would be more appropriate in my opinion. A similar logic follows for `generate_components` method in the `InVesselBuild` class.